src/safe_apps/serializers.py

```diff
@@ -30,21 +32,33 @@ class NoRestrictionsAccessControlPolicySerializer(serializers.Serializer[SafeApp
     )
 
 
-class TagSerializer(serializers.Serializer[Tag]):
-    def to_representation(self, instance: Tag) -> str:
+class TagSerializer(serializers.ModelSerializer[Tag]):
+    class Meta:
+        fields = "__all__"
+        model = Tag
+        ref_name = "safe_apps.serializers.TagSerializer"
+
+    def to_representation(self, instance: Tag) -> str:  # type: ignore[override]
         return instance.name
 
 
-class FeatureSerializer(serializers.Serializer[Feature]):
+class FeatureSerializer(serializers.ModelSerializer[Feature]):
     class Meta:
+        fields = "__all__"
```

Comment:
Similar to the above:

```python
fields = ['key']
```

Reply:
Fixed in e2f65178e8a897190d0620c758e5c9e16a8ada4a.